writehat/templates/panes/engagementReportsList.html

```diff
@@ -4,27 +4,30 @@
       <h4>Reports: {{ engagement.reports|length }}</h4>
     </div>
     <div class='ml-auto'>
+      {% include 'snippets/statusButtonGroup.html' %}
       {% include 'snippets/textButton.html' with name='New Report' id='reportCreate' type='plus' class='btn-primary' %}
     </div>
   </div>
-
-  <table class="writehat-table container-fluid table-light table-striped table-hover overflow-auto paginate writehat-reports">
+  <table class="writehat-table container-fluid table-light table-striped table-hover overflow-auto paginate writehat-reports" id="reports">
     <thead>
+      <th></th>
       <tr>
         <th><strong>Name</strong></th>
         <th><strong>Components</strong></th>
         <th><strong>Created</strong></th>
         <th><strong>Modified</strong></th>
+        <th><strong>Status</strong></th>
         <th></th>
       </tr>
     </thead>
     <tbody class="text-nowrap">
       {% for report in engagement.reports|dictsortreversed:"modifiedDate" %}
-        <tr report-id="{{ report.id }}" report-name="{{ report.name }}">
+        <tr class="report-row" report-id="{{ report.id }}" report-name="{{ report.name }}">
           <td><a href="/engagements/report/{{ report.id }}/edit">{{ report.name }}</a></td>
           <td><a href="/engagements/report/{{ report.id }}/edit">{{ report.numComponents }}</a></td>
           <td><a href="/engagements/report/{{ report.id }}/edit">{{ report.createdDate }}</a></td>
           <td><a href="/engagements/report/{{ report.id }}/edit">{{ report.modifiedDate }}</a></td>
+          <td><a href="/engagements/report/{{ report.id }}/edit">{{ report.status }}</a></td>
```

Comment:
In the table, can we render the status' label instead of the internal value? (e.g. 'Active' instead of 'active')